test/test_utils_log.cpp

```diff
@@ -0,0 +1,380 @@
+// Copyright (C) 2024 Intel Corporation
```

Comment:
pls drop `test_` from the name of the file

Reply:
some files has test prefix, some doesn't - what is a rule here ( btw. I prefer names with test_prefix so you can distinguish it from file with implementation, without checking directories)

Reply:
we have a mess in tests files - IMO we could leave it and do the cleanup in a separate PR